slices/mysql-common.yaml

```diff
@@ -0,0 +1,14 @@
+package: mysql-common
+
+essential:
+  - mysql-common_copyright
+
+slices:
+  config:
+    contents: 
+      /etc/mysql/conf.d/mysql.cnf:
+      /etc/mysql/conf.d/mysqldump.cnf:
+      /etc/mysql/my.cnf.fallback:
+  copyright:
```

Comment:
Let's put an empty line before the `copyright` line to match style of other SDFs.